tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/get-started-experience.cy.js

```diff
@@ -0,0 +1,44 @@
+// <reference types="Cypress" />
+
+describe('Start Setup WP Experience Page', function () {
+
+	before(() => {
+		cy.visit('wp-admin/?page=nfd-onboarding&flow=ecommerce#/wp-setup/step/get-started/experience');
+		cy.injectAxe();
+	});
+
+    it('Check if Drawer Suppressed', () => {
+        cy.get('.nfd-onboarding-drawer__panel-site-title-container').scrollIntoView().should('not.be.visible');
+    });
+
+    it('Check if Headers Load', () => {
+        cy.get('.nfd-step-card-heading').should('exist');
+        cy.get('.nfd-step-card-subheading-other').should('exist');
+        cy.get('.nfd-step-card-question').should('exist');
+    });
+
+    it('Check if radio options load', () => {
+        cy.get('.components-radio-control__option').should('exist');
```

Comment:
This being there does not mean that the `input` is present, please check my tax info PR I've added a better checks for radio buttons there.